fiatconnect-api.md

```diff
@@ -1796,6 +1796,897 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccountIfsc`
+
+`CheckingAccountIfsc` is a Fiat Account schema that represents accounts for transactions across `INR` currency.
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "ifsc": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIfsc"
+}
+```
+####  7.3.2.3. `CheckingAccountIban`
+
+`CheckingAccountIban` is a Fiat Account schema that represents accounts for transactions across `EUR` currency.
+
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "iban": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIban"
+}
+```
+
+####  7.3.2.4. `CheckingAccountBankCode`
+
+`CheckingAccountBankCode` is a Fiat Account schema that represents accounts for transactions across `NG`, `GH`, `KE`, `ZA`, `PH`, `SN`, `CN`, `GB`, `SG`, `TZ`, `HK` currencies.
```

Comment:
Aren't these countries, not currencies? 

Also, there's many countries which use checking accounts... not sure the list of countries where this fiat account schema is accepted by BitSSA belongs here

Reply:
@cajubelt yes these are countries and the CheckingAccountBankCode schema is supported in these countries only.

Reply:
@cajubelt - fixed